src/Application/Renderer/Template/Action/SourcecodeHandler.php

```diff
@@ -37,7 +37,7 @@ public function __construct(Analyzer $analyzer)
      */
     public function __invoke(Action $action)
     {
-        $artifact = $action->getRenderPass()->getDestination() . '/' . ltrim($action->getDestination(), '\\/');
+        $artifact = $action->getRenderContext()->getDestination() . '/' . ltrim($action->getDestination(), '\\/');
```

Comment:
Bugger! I thought that was the British spelling. Nicely noticed. We should then use artifact I guess :)